src/components/RuleBuilder/utils/export.js

```diff
@@ -11,40 +11,25 @@ export const exportRuleBuilder = (
   groups: exportGroups(groups, combinedRuleOptions, absoluteRuleOptions)
 })
 
-const exportGroups = (groups, combinedRuleOptions, absoluteRuleOptions) =>
+const exportGroups = (groups, absoluteRuleOptions) =>
   groups.map(({ not, absolute, combined, combinator, advancedTargeting }) => ({
     not,
     combinator,
-    combined: exportRules(combined, combinedRuleOptions),
+    combined: exportRules(combined),
     absolute: advancedTargeting
       ? exportRules(absolute, absoluteRuleOptions)
```

Comment:
Extra argument for `exportRules`

Reply:
On line 20

Reply:
9abaf5f6eb3fa71d22afef8c09c7ea513526e2e8